tests/files/src_test_gen.py

```diff
@@ -0,0 +1,63 @@
+import subprocess
+import shutil
+from pathlib import Path
+from typing import List
+
+from pydantic import BaseModel, TypeAdapter
+
+
+class OTUContents(BaseModel):
+    taxid: int
+    otu_schema: list[str]
+    contents: list[str]
+
+
+ref_contents_adapter = TypeAdapter(List[OTUContents])
```

Comment:
Need a better name and a docstring.

Reply:
Unsure what a good replacement would be, but replaced for now.